### PR TITLE
Bump github.com/gogo/protobuf to v1.3.2 in .bingo

### PR DESCRIPTION
**Pull request by simonpasquier**
* [ ] I added CHANGELOG entry for this change.
* [X] Change is not relevant to the end user.

## Changes

In practice, the recent bump of github.com/gogo/protobuf (introduced by #3804) changes the generated code to fix [CVE-20213121](https://nvd.nist.gov/vuln/detail/CVE-2021-3121).

## Verification

No functional change.

### CODE REVIEW

File: .github/workflows/go.yaml [hunk 1]
```diff
@@ -54,6 +54,33 @@ jobs:
...
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+
+      - name: Generate protobuf code
+        run: make proto
+
+      - name: Format
+        run: make format
+
+      - name: Format
```
Comment (prmsrswt) [inline .github/workflows/go.yaml:82]:
```suggestion
      - name: Check proto
```

Nit: Something like this?

**Review by prmsrswt — APPROVED** [review]
LGTM! I just have a small nit but not a blocker.

**Review by bwplotka — APPROVED** [review]
Thanks!

File: .github/workflows/go.yaml [hunk 2]
```diff
@@ -54,6 +54,33 @@ jobs:
...
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15.x
+
+      - uses: actions/cache@v1
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+
+      - name: Generate protobuf code
```
Comment (bwplotka) [inline .github/workflows/go.yaml:76]: Hm, why not just adding proto to `lint` Makefile and... done? 🤔

**Review by squat — APPROVED** [review]
v nice :) thanks!

File: .github/workflows/go.yaml (hunk 2 above)
Comment (simonpasquier) [reply .github/workflows/go.yaml:76]: whatever works for you :)

Comment (simonpasquier) [reply .github/workflows/go.yaml:76]: wait it's already done!

File: .github/workflows/go.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ name: go
 on:
   push:
     branches:
-      - main
+      - master
```
Comment (kakkoyun) [inline .github/workflows/go.yaml:6]: Haven't we just renamed `master` to `main`? I guess this change shouldn't be here.

**Review by kakkoyun — CHANGES_REQUESTED** [review]
Looks great! Only one problem I have seen.

File: .github/workflows/go.yaml (hunk 3 above)
Comment (simonpasquier) [reply .github/workflows/go.yaml]: good catch! fixed.

**Review by kakkoyun — APPROVED** [review]

**Comment by codefromthecrypt** [comment]
FYI: I think you accidentally missed .bingo/Variables.mk and variables.env

**Comment by simonpasquier** [comment]
@codefromthecrypt correct although it doesn't change anything in practice: the code has been generated with gogo/protobuf v1.3.2. I won't have time to submit a follow-up PR so go ahead if you wish :)

**Comment by codefromthecrypt** [comment]
@simonpasquier sure why not. #3902

**Pull request merged**